Team6220_2021/src/main/java/org/firstinspires/ftc/team6220_2021/AutonomousCompetition.java

```diff
@@ -0,0 +1,23 @@
+package org.firstinspires.ftc.team6220_2021;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+
+@Autonomous(name = "Autonomous Competition", group = "Autonomous")
+public class AutonomousCompetition extends MasterAutonomous {
+
+    @Override
+    public void runOpMode() {
+
+        Initialize();
+        waitForStart();
+
+        telemetry.addData("hi", imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES).firstAngle);
```

Comment:
Do you want to include a more useful string than "hi"? Maybe explain what the value you're printing out is